opa/decisionlog/decision_log.go

```diff
@@ -21,15 +20,10 @@ func (e *internalError) Error() string {
 }
 
 // LogDecision - Logs a decision log event
-func LogDecision(ctx context.Context, manager *plugins.Manager, info *server.Info, result *envoyauth.EvalResult, err error) error {
```

Comment:
Technically this is a breaking change. Someone could be using this function from elsewhere.

The "fix" would be to add another static function here and keep the old one as-is. **But** I think it's _very unlikely_ that someone cares for this function, so let's not be too timid about it. :+1:

Reply:
[Skipper](https://github.com/zalando/skipper) is using the function. It would have been nice to have a note in the release note mentioning it's a breaking change. Still since it's 1.0.0 breaking changes were expected.